laituri/docker/credential_manager/docker_v1.py

```diff
@@ -17,7 +18,8 @@ def docker_v1_credential_manager(
     registry_credentials: Dict,
     log_status: Callable
 ):
-    domain = image.split('/')[0]
+    # If image looks like a dockerhub image, use docker.io as the registry domain
+    domain = image.split('/')[0] if image_has_domain(image) else 'docker.io'
```

Comment:
Maybe this should be `get_image_domain()`, since you now have this incantation twice in the codebase..?

```python
def get_image_domain(image: str) -> str:
    if image.count("/") > 1:
       return image.split("/")[0]
    return "docker.io"
```
or thereabouts? :)